src/tap_reporter.wren

```diff
@@ -6,47 +6,54 @@ StackTraceReport.withoutColors
 Expect.withoutColors
 
 class TapReporter {
+    // https://testanything.org/tap-version-13-specification.html
+    static tapVersion { 13 }
+
     construct new(name) {
         _name = name
         _fail = _skip = _success = _number = 0
         _plan = false
+        _section = ""
     }
     start() {
-        System.print("TAP version 13")
+        System.print("TAP version %(this.type.tapVersion)")
     }
     start(num_tests) {
         start()
         _plan = true
         System.print("1..%(num_tests)")
     }
+    section(name) {
+        _section = "%(name): "
+    }
     skip(name) {
         _skip = _skip + 1
         _number = _number + 1
-        System.print("ok %(_number) %(name) # SKIP")
-    }
-    section(name) {
-        System.print("## %(name.trim())")
+        System.print("ok %(_number) %(_section)%(name) # SKIP")
     }
     success(name) {
         _success = _success + 1
         _number = _number + 1
-        System.print("ok %(_number) %(name)")
+        System.print("ok %(_number) %(_section)%(name)")
     }
     fail(name, fiber) {
         _fail = _fail + 1
         _number = _number + 1
-        System.print("not ok %(_number) %(name)")
+        System.print("not ok %(_number) %(_section)%(name)")
         for (line in fiber.error.toString.split("\n")) {
-            System.print("# %(line)")
+            System.print(line.isEmpty ? "#" : "# %(line)")
         }
         if (Capabilities.hasMirror) {
             System.print("#")
             var st = StackTraceReport.new(fiber)
             for (line in st.toString.split("\n")) {
-                System.print("# %(line)")
+                System.print(line.isEmpty ? "#" : "# %(line)")
```

Comment:
Why not just `trim()` if you case about whitespace?

Reply:
If line is empty, I don't want to print a dangling space after the hash. It's a nitpicky little thing.

Reply:
Right, hence use `trim()`... no need for a ternary.

Reply:
I'm not sure I follow. Do you mean
```
System.print("# %(line)".trim())
```

Reply:
Yes, exactly. OR perhaps make a local `print` for TAP reporter that has this built-in... and use that everywhere in TAP.

Reply:
Done.